src/main/java/co/rsk/bitcoinj/core/TransactionWitness.java

```diff
@@ -46,6 +58,64 @@ public static TransactionWitness createWitness(@Nullable final TransactionSignat
         return witness;
     }
 
+    public static TransactionWitness createWitnessErpStandardScript(Script witnessScript, List<TransactionSignature> signatures) {
+        List<byte[]> pushes = new ArrayList<>(signatures.size() + 3);
```

Comment:
Let's add a comment about why +3.
First OP_0, OP_NOTIF argument and redeem script, right?